client/src/assets/scss/theme.scss

```diff
@@ -46,11 +46,23 @@ $BCgovAGold2: #FFF8EF;
 $BCgovAGold3: #FFE0BC;
 $BCgovAGold4: #FFD4A2;
 
+// Link
+$link: #1669bb;
+
+// Approved
+$approved: #1a9031;
+
+// Caution
+$caution: #bf4803;
+
 // Error Colors
 $BCgovInputError: #ff5252;
+$error: #d3272c;
 
 // Form Inputs
 $BCgovInputBG: #ffffff;
 
 // Font Colors
 $BCgovFontColorInverted: #ffffff;
+$dk-text: #212529;
+$text: #495057;
```

Comment:
Ah, instead of re-declaring these colours, we should use the existing named colours (in this case, $gray9 and $gray7).

Reply:
Doh
Quick fix. Will do!